site/docs/v1/tech/apis/_key-generate-post-request-body.adoc

```diff
@@ -14,6 +14,9 @@ The algorithm used to generate the Key. The following values represent algorithm
 * `HS384` - HMAC using SHA-384 hash algorithm
 * `HS512` - HMAC using SHA-512 hash algorithm
 
+[field]#key.issuer# [type]#[String]# [optional]#Optional#::
+The issuer of the RSA or EC certificate. This field is omitted for HMAC key types.
```

Comment:
Since this is a request body doc - the term omit doesn't seem correct. Do you mean it is not utilized for HMAC keys and should be omitted from the request? What happens if I set it for an HMAC key?

Reply:
It is ignored for HMAC 😄